interface/connection/conn.go

```diff
@@ -4,55 +4,97 @@
 package connection
 
 import (
+	"fmt"
 	"io"
 	"net"
+	"os"
 	"sync"
 )
 
+const defaultUnixSocketAddress = "/run/parsec/parsec.sock"
+const parsecEndpointEnvironmentVariable = "PARSEC_SERVICE_ENDPOINT"
```

Comment:
Could you please add a section in the README.md indicating this behaviour? Similarly as we have on [the Rust client](https://github.com/parallaxsecond/parsec-client-rust#locating-the-parsec-endpoint)